src/components/page-replica/index.js

```diff
@@ -104,7 +104,7 @@ const PageReplica = (props) => {
                   </React.Fragment>
                 )}
                 {selectedReplica &&
-                  selectedReplica.restartCount != NaN &&
+                  Number.isNaN(selectedReplica.restartCount) &&
```

Comment:
Should it be `!Number.isNaN(selectedReplica.restartCount)` ?

Reply:
Of course. Good catch!